packages/cli/src/models/network/TransactionController.ts

```diff
@@ -1,8 +1,12 @@
-import { Transactions, Logger, ZWeb3, TxParams } from 'zos-lib';
+import { Transactions, Logger, ZWeb3, TxParams, ABI } from 'zos-lib';
 import { isValidUnit, prettifyTokenAmount, toWei, fromWei } from '../../utils/units';
 import { ERC20_PARTIAL_ABI } from '../../utils/constants';
 import { allPromisesOrError } from '../../utils/async';
+import LocalController from '../local/LocalController';
+import ZosPackageFile from '../files/ZosPackageFile';
+import ZosNetworkFile from '../files/ZosNetworkFile';
 
+const { buildCallData, callDescription }  = ABI;
```

Comment:
```suggestion
const { buildCallData, callDescription } = ABI;
```

Reply:
As you said in the past, we should definitely add linter rules for these :joy:

Reply:
https://github.com/zeppelinos/zos/issues/849